google-cloud-datastore/src/test/java/com/google/cloud/datastore/testing/ITLocalDatastoreHelperTest.java

```diff
@@ -54,6 +57,44 @@ public void testCreate() {
     assertTrue(helper.getProjectId().startsWith(PROJECT_ID_PREFIX));
   }
 
+  @Test
+  public void testCreateWithBuilder() {
+    LocalDatastoreHelper helper =
+        LocalDatastoreHelper.newBuilder()
+            .setConsistency(0.75)
+            .setPort(0)
+            .setStoreOnDisk(false)
+            .setDataDir(DATA_DIR)
+            .build();
+    assertTrue(Math.abs(0.75 - helper.getConsistency()) < TOLERANCE);
+    assertTrue(helper.getProjectId().startsWith(PROJECT_ID_PREFIX));
+    assertFalse(helper.isStoreOnDisk());
+    assertEquals(0, helper.getPort());
+    assertEquals(DATA_DIR, helper.getGcdPath());
+    helper = LocalDatastoreHelper.newBuilder().build();
+    assertTrue(Math.abs(0.9 - helper.getConsistency()) < TOLERANCE);
+    assertTrue(helper.getProjectId().startsWith(PROJECT_ID_PREFIX));
+  }
+
+  @Test
+  public void testCreateWithToBuilder() {
+    LocalDatastoreHelper helper =
+        LocalDatastoreHelper.toBuilder()
+            .setConsistency(0.75)
+            .setPort(0)
+            .setStoreOnDisk(false)
+            .setDataDir(DATA_DIR)
+            .build();
+    assertTrue(Math.abs(0.75 - helper.getConsistency()) < TOLERANCE);
+    assertTrue(helper.getProjectId().startsWith(PROJECT_ID_PREFIX));
+    assertFalse(helper.isStoreOnDisk());
+    assertEquals(0, helper.getPort());
+    assertEquals(DATA_DIR, helper.getGcdPath());
+    helper = LocalDatastoreHelper.toBuilder().build();
+    assertTrue(Math.abs(0.9 - helper.getConsistency()) < TOLERANCE);
+    assertTrue(helper.getProjectId().startsWith(PROJECT_ID_PREFIX));
```

Comment:
helper = LocalDatastoreHelper.toBuilder().build();
=>
helper = helper.toBuilder().build();

you need to check that no property has changed